tests/sentry/integrations/github/test_webhooks.py

```diff
@@ -134,6 +165,54 @@ def test_simple(self, mock_get_jwt):
         assert commit.author.external_id is None
         assert commit.date_added == datetime(2015, 5, 5, 23, 40, 15, tzinfo=timezone.utc)
 
+    @patch("sentry.integrations.github.client.get_jwt")
+    def test_auto_linking_missing_feature_flag(self, mock_get_jwt):
+        mock_get_jwt.return_value = ""
+
+        project = self.project  # force creation
```

Comment:
how does this force creation?

Reply:
i copy pasted this from a different function. i guess if you call self.project it creates a project too